sources/packages/backend/workflow/test/2023-2024/parttime-assessment/disbursements/parttime-assessment-2023-2024-disbursements.e2e-spec.ts

```diff
@@ -0,0 +1,265 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeConfigureDisbursement,
+} from "../../../test-utils";
+import { addDays, getISODateOnlyString, getUTCNow } from "@sims/utilities";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-disbursements`, () => {
+  const partTimeDisbursementData = {
```

Comment:
Please create a method similar to `createFakeAssessmentConsolidatedData`. This would support the recommendation from this comment: https://github.com/bcgov/SIMS/pull/2788/files#r1480390753